Dockerfile

```diff
@@ -23,6 +23,10 @@ RUN npm ci --omit=optional
 
 RUN npm run build
 
+# Install only production dependencies
+# This is needed to remove the dev dependencies that were installed in the previous step
+RUN cd backend && npm prune --omit=dev
```

Comment:
The goal is to not copy over the build tooling while building images, but post building the source?

You cut out the omit optional, cache clean, & the chmod effort. The drop of the omit optional & I guess the cache clean are done in the lines above 🤔 But what was the value of the `chmod`?

Reply:
I'm glad that you asked, I didn't add that into the description, apologies, let me add more context.
That change was done here https://github.com/opendatahub-io/odh-dashboard/pull/1256 due to issues with the default user and openshift user's randomness.
We already fixed that back when we introduced user id's, you can check that is fixed by first running an old version:
`podman run --rm --user 65536 quay.io/opendatahub/odh-dashboard:v2.9.2 ``
Which will induce the error
And then do the exact same thing with this current build:
`podman run --rm --user 65536 quay.io/lferrnan/odh-dashboard:fix-dockerfile`